webapp/src/types/model.ts

```diff
@@ -106,3 +106,42 @@ export type ChannelSubscription = {
     filters: ChannelSubscriptionFilters;
     name: string;
 }
+
+// Collection of ChannelSubscriptions
+export type ChannelSubscriptions = {[subscriptionId: string]: ChannelSubscription};
+
+export enum JiraInstanceType {
+    SERVER = 'server',
+    CLOUD = 'cloud',
+}
+
+export type CreateModal = {
+    postId?: string;
+    description?: string;
+    channelId?: string;
```

Comment:
Looks like there is now a duplicate definition of this with `CreateModalValue`. @AGMETEOR Can you please remove this one, instead preferring `CreateModalValue`?

Reply:
@mickmister removed the duplicate type